app/services/pipeline_report_service.rb

```diff
@@ -241,7 +247,11 @@ def fetch_taxon_counts(_pipeline_run_id, _background_id)
                                          tax_level: [TaxonCount::TAX_LEVEL_SPECIES, TaxonCount::TAX_LEVEL_GENUS]
                                        )
                                        .where.not(
-                                         tax_id: [TaxonLineage::BLACKLIST_GENUS_ID, TaxonLineage::HOMO_SAPIENS_TAX_ID]
+                                         tax_id: [
+                                           TaxonLineage::BLACKLIST_GENUS_ID,
+                                           TaxonLineage::HOMO_SAPIENS_TAX_ID,
```

Comment:
It looks like `HUMAN_TAX_IDS` ([9605, 9606]) includes `HOMO_SAPIENS_TAX_ID` (9606), is there a reason why we have both?

Reply:
Actually no. Should have only HUMAN_TAX_IDS. Good catch.

Reply:
Will change `HOMO_SAPIENS_TAX_ID` to the correct definition. We should deprecate `HUMAN_TAX_IDS`.